Softeq.NetKit.Components.EventBus.Service/Connection/ServiceBusQueueConnection.cs

```diff
@@ -52,7 +52,7 @@ public async ValueTask DisposeAsync()
 
         private static IQueueClient CreateClient(string connectionString, string queueName)
         {
-            var client = new QueueClient(connectionString, queueName, ReceiveMode.PeekLock, RetryPolicy.Default);
+            var client = new QueueClient(connectionString, queueName, ReceiveMode.ReceiveAndDelete, RetryPolicy.Default);
```

Comment:
It seems that the message can be lost for ReceiveMode.ReceiveAndDelete in case a fatal error happen just after message was received by the client. Probably it's better to keep it as ReceiveMode.PeekLock and complete in finally block just message handler processing executed
```csharp
try
{
    Handle(message);
}
finally
{
   message.Complete();
}
```